cognitedata/reveal
# fix(react-components): search hook bad request and duplicate views

#### Type of change

![Bug](https://img.shields.io/badge/Type-Bug-red) 

## Description :pencil:


## How has this been tested? :mag:


In searchHooks story.


## Test instructions :information_source:


1. Go to searchHooks story
2. Search for "DHAJDKFHAJSKDFHAS"
3. Observe no error in the console

## Checklist :ballot_box_with_check:
